lib/rules/match-exported.js

```diff
@@ -24,6 +27,25 @@ function getStringToCheckAgainstExport(parsed) {
     }
 }
 
+function transform(exportedName, context) {
```

Comment:
What about:

```javascript
/// ...
var transforms = {
    kebab: kebabCase,
    snake: snakeCase,
    camel: camelCase
};
// ...

function transform(exportedName, transformName) {
    var transform = transforms[transformName];

    return transform ? transform(exportedName) : exportedName;
}
```

This way the exported name can always be passed to this function regardless of wether a transform option is actually passed.